extensions/amp-story/0.1/amp-story-hint.js

```diff
@@ -114,14 +117,17 @@ const FIRST_PAGE_NAVIGATION_OVERLAY_TIMEOUT = 275;
  * User Hint Layer for <amp-story>.
  */
 export class AmpStoryHint {
-
   /**
    * @param {!Window} win
+   * @param {!Element} storyElement Element where to append the bookend
    */
-  constructor(win) {
+  constructor(win, storyElement) {
```

Comment:
nit: prefer something like `parentElement` or `containerElement` to imply that this implementation does not particularly care that the element passed is a story

Reply:
Replaced here and in ``bookend.js``